.buildkite/full/react-native-ios-pipeline.full.yml

```diff
@@ -340,48 +352,75 @@ steps:
+      # causing a build failure - see https://github.com/wix/react-native-navigation/issues/7821
+      # TODO: Investigate and try to re-enable when we add tests for more recent React Native versions
+      - label: ":bitbar: :mac: react-native-navigation {{matrix}} iOS 16 (Old Arch) end-to-end tests"
+        depends_on: "build-react-native-navigation-ios-fixture-old-arch"
         timeout_in_minutes: 60
         plugins:
-          artifacts#v1.5.0:
-            download: "build/r_navigation_0.69.ipa"
+          artifacts#v1.9.0:
+            download: "test/react-native/features/fixtures/generated/react-native-navigation/old-arch/{{matrix}}/output/reactnative.ipa"
             upload: ./test/react-native/maze_output/**/*
           docker-compose#v4.12.0:
             pull: react-native-maze-runner
             run: react-native-maze-runner
-            use-aliases: true
+            service-ports: true
             command:
-              - --app=build/r_navigation_0.69.ipa
-              - --farm=bs
+              - --app=/app/features/fixtures/generated/react-native-navigation/old-arch/{{matrix}}/output/reactnative.ipa
+              - --farm=bb
               - --device=IOS_16
               - --a11y-locator
               - --fail-fast
-              - features/navigation.feature
-        concurrency: 5
-        concurrency_group: "browserstack-app"
+              - --no-tunnel
+              - --aws-public-ip
+              - features/react-native-navigation.feature
+        retry:
+          manual:
+            permit_on_passed: true
+        env:
+          RN_VERSION: "{{matrix}}"
+          RCT_NEW_ARCH_ENABLED: "0"
+          REACT_NATIVE_NAVIGATION: "true"
+        concurrency: 25
+        concurrency_group: "bitbar"
         concurrency_method: eager
+        matrix:
+          - "0.71"
+          - "0.72"
 
-      # See: PLAT-5173
-      - label: ":browserstack: :ios: react-native-navigation 0.66 iOS 16 end-to-end tests"
-        skip: "See PLAT-5173"
-        depends_on: "react-native-navigation-0-66-ipa"
-        timeout_in_minutes: 60
-        plugins:
-          artifacts#v1.5.0:
-            download: "build/r_native_navigation_0.66.ipa"
-            upload: ./test/react-native/maze_output/**/*
-          docker-compose#v4.12.0:
-            pull: react-native-maze-runner
-            run: react-native-maze-runner
-            use-aliases: true
-            command:
-              - --app=build/r_native_navigation_0.66.ipa
-              - --farm=bs
-              - --device=IOS_16
-              - --a11y-locator
-              - --fail-fast
-              - features/navigation.feature
-        concurrency: 5
-        concurrency_group: "browserstack-app"
-        concurrency_method: eager
+      # Skipped due to build failures with react-native-navigation when new arch is enabled
+      # TODO: Investigate and try to re-enable when we add tests for more recent React Native versions
```

Comment:
prefer raising a ticket and adding `skip: "See PLAT-1234"` than commenting out

Reply:
Discussed this with @tomlongridge and we decided not to raise a ticket for this because it's not something we can fix on our side - instead we'll re-assess when we come to add the next RN version to the test matrix.